src/MagentoHackathon/Composer/Magento/Plugin.php

```diff
@@ -200,6 +214,9 @@ public function onNewCodeEvent(Event $event)
         $magentoModules = array_filter(
             $this->composer->getRepositoryManager()->getLocalRepository()->getPackages(),
             function (PackageInterface $package) use ($packageTypeToMatch) {
+                if ($package instanceof AliasPackage) {
+                    return false;
```

Comment:
No idea what this logic does, but you sure this wasn't supposed to be `continue` instead?